<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dividir PDF</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_pdf">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>10</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cargar PDF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pdf">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>17</y>
      <width>410</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ubicación del pdf</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cant_a_partir">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cant_pag">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>53</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pag_a_recortar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Páginas a recortar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_a_partir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>330</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A partir de cual página va a recortar</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cant_pag_a_recortar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_iniciar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Iniciar recorte</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>El PDF contiene</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>52</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>páginas</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pag_a_recortar_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Intérvalo</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cant_intervalo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pag_a_recortar_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre de la carpeta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_carpeta">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nombre de la carpeta sin simbolos</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
